BE/src/main/java/kr/codesquad/issuetracker/domain/comment/Comment.java

```diff
@@ -0,0 +1,41 @@
+package kr.codesquad.issuetracker.domain.comment;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import kr.codesquad.issuetracker.domain.BaseTimeEntity;
+import kr.codesquad.issuetracker.domain.image.CommentImage;
+import kr.codesquad.issuetracker.domain.issue.Issue;
+import kr.codesquad.issuetracker.domain.member.Member;
+
+@Entity
+@Table(name = "comments")
+public class Comment extends BaseTimeEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "comment_id")
+	private Long id;
+
+	private String content;
+
+	@OneToMany(mappedBy = "comment")
+	private List<CommentImage> commentImages = new ArrayList<>();
+
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "member_id")
+	private Member member;
```

Comment:
작성자를 명확하게 뜻하는 변수명을 사용하는건 어떨까요?
```suggestion
	private Member writer;
```

Reply:
좋은 의견 감사드립니다 ㅎㅎ 바로 수정하겠습니다 :)